pkg/models/action.go

```diff
@@ -11,6 +11,8 @@ import (
 type ActionType string
 
 const (
+	// ActionTypeREQUESTEDITS captures enum value REQUEST_EDITS (for admin actions v2)
+	ActionTypeREQUESTEDITS ActionType = "REQUEST_EDITS"
```

Comment:
I think this can be modified when unifying with Dylan's PR, I like how Dylan separated the new actions in their own declaration block.

Reply:
I can move this up into the new declaration block after this PR is merged, when reconciling `main` with my branch.

Reply:
I'll go ahead and move it.

Reply:
resolved in [c38f771](https://github.com/CMSgov/easi-app/pull/2129/commits/c38f7719b2cd5c17d9a5d3b6c1b2479fef146da5)